docs/AccelNNPAHowToUseAndTest.md

```diff
@@ -0,0 +1,29 @@
+<!--- SPDX-License-Identifier: Apache-2.0 -->
+
+# Build and test for Accelerator NNPA
+
+## Build
+
+The following CMake variable is required to build onnx-mlir for NNPA.
+
+- `-DONNX_MLIR_ACCELERATORS=NNPA`
+
+## Test
+
+Lit tests and numerical tests are provided for NNPA.
+
+- Lit tests
+
+When building onnx-mlir for NNPA, lit tests for NNPA also run with the command for CPU. The lit tests for NNPA are included in `test/mlir/accelerators/nnpa`.
+
+```
+cmake --build . --target check-onnx-lit
+```
+
```

Comment:
Can you confirm, and add a sentence to the extend that literal tests of the NNPA can be always performed on any platforms, regardless of whether NNPA is actually present or not. 

@gongsu832 can you remind us on how we can add this test on non-z16 machines? (just for the lit test). Thanks

Reply:
@AlexandreEichenberger Yes. The lit tests of the NNPA can run on non-z16 machines when onnx-mlir is built with  `-DONNX_MLIR_ACCELERATORS=NNPA`.

Reply:
NNPA is built on Z because we need Z to build libzdnn. NNPA lit tests is currently run on Jenkins s390x where some of the tests, e.g. compiled time stickifies, are related to big endian.